Pipfile.lock

```diff
@@ -1,7 +1,20 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "d797e580ddcddc99bf058109ab0306ad584c2902752a3d4076ba713fdc580fb7"
+            "sha256": "2cca231e2bd4b5fdaa06fd33b68143231524f2855847ebf4cc497462e135fdf2"
+        },
+        "host-environment-markers": {
```

Comment:
Huh. Where'd this come from? Also, why did you sort all the hashes in this file?

Reply:
this happened automatically when I installed the `titlecase` package